### PR TITLE
feat: add CSP support

### LINKED ISSUE #9: CSP support
Currently, in our private core framework, we use internal CSP package which wraps [express-csp-header](https://github.com/frux/csp/tree/master/packages/express-csp-header) adding some common-used internal csp rules.

We should implement CSP support in ExpressKit using express-csp-header, and:

1. Check if there is any additional differences between express-csp-header and our internal package that we should implement here (except for included default rules)
2. Understand how can we continue define/extend ExpressKit CSP rules using that internal package in our apps

### PR DESCRIPTION
**Pull request by ValeraS**
Closes #9

### CODE REVIEW

**Comment by melikhov-dev** [comment]
It's ok, but I want to integrate it to the https://github.com/datalens-tech/datalens-ui and check.

**Review by melikhov-dev — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -26,3 +26,32 @@ app.run();
...
+## CSP
+
+`config.ts`
+
+```typescript
+import type {AppConfig} from '@gravity-ui/nodekit';
+import {csp} from '@gravity-ui/expresskit';
+
+const config: Partial<AppConfig> = {
+    // cspDisable: true,
```
Comment (melikhov-dev) [inline README.md:39]: Why not disable by default?

Comment (resure) [reply README.md:39]: Yeah, let's change default to off?

> replace cspDisable with cspEnable, if we don't want to force users to use it.

Looks good

File: src/types.ts [hunk 2]
```diff
@@ -56,6 +60,14 @@ declare module '@gravity-ui/nodekit' {
         appAfterAuthMiddleware?: RequestHandler[];
 
         appTelemetryChEnableSelfStats?: boolean;
+
+        cspEnable?: boolean;
```
Comment (resure) [inline src/types.ts:64]: Let's add `express` prefix to this options

File: README.md [hunk 3]
```diff
@@ -26,3 +26,32 @@ app.run();
...
+## CSP
+
+`config.ts`
+
+```typescript
+import type {AppConfig} from '@gravity-ui/nodekit';
+import {csp} from '@gravity-ui/expresskit';
+
+const config: Partial<AppConfig> = {
+    cspEnable: true,
```
Comment (resure) [inline README.md:39]: missing prefix here too

**Review by resure — APPROVED** [review]

**Pull request merged**